.github/workflows/refapp-2x-patient-allergies.yml

```diff
@@ -0,0 +1,40 @@
+name: RefApp 2.x Patient Allergies
```

Comment:
what do you think of `refapp-2x-patient-allergies.yml` vs `refapp-2x-allergy.yml`

Reply:
@Irenyak1 did you take a look at the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/298/files#r778104882)?

Reply:
@Irenyak1 did you take a look at the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/298/files#r778104882)?

Reply:
@kdaud honestly if the final decision depended on me. I would leave them as  refapp-2x-patient-allergies.yml and  RefApp 2.x Patient Allergies. In my opinion it makes more sense if they are labelled this way.

Reply:
@Irenyak1 your final decision is honored !!!